doc/release/release_0.16.rst

```diff
@@ -0,0 +1,389 @@
+Announcement: scikit-image 0.16.0
+=================================
+
+We're happy to announce the release of scikit-image v0.16.0!
+
+scikit-image is an image processing toolbox for SciPy that includes algorithms
+for segmentation, geometric transformations, color space manipulation,
+analysis, filtering, morphology, feature detection, and more.
+
+For more information, examples, and documentation, please visit our website:
+
+https://scikit-image.org
+
+Starting from this release, scikit-image will follow the spirit of the recently
+introduced NumPy deprecation policy, `NEP 29
+<https://github.com/numpy/numpy/blob/master/doc/neps/nep-0029-deprecation_policy.rst>__`.
+Accordingly, scikit-image 0.16 drops support for Python 3.5.
+This release of scikit-image officially supports Python 3.6 and 3.7.
+
+Special thanks to Matthias Bussonnier for `Frappuccino
+<https://github.com/Carreau/frappuccino>`__, which helped us catch all API
+changes and nail down the APIs for new features.
+
+New Features
+------------
+- Introduced a new `skimage.evaluate` module containing simple metrics (mse,
+  nrme, psd) and segmentation metrics (adapted rand error, variation of
+  information) (#4025)
+- Added n-dimensional TV-L1 optical flow algorithm for registration --
+  `skimage.registration.optical_flow_tvl1` (#3983)
+- Added a function drawing a line in n-dimensional space -- `skimage.draw.line_nd`
+  (#2043)
+- Added 2D Farid & Simoncelli edge filters - `skimage.filters.farid`,
+  `skimage.filters.farid_h`, and `skimage.filters.farid_v` (#3775)
+- Added 2D majority voting filter assigning to each pixel the most commonly
+  occurring value within its neighborhood -- `skimage.filters.majority`
+  (#3836, #3839)
+- Added multi-level threshold "multi-Otsu" method, a thresholding algorithm
+  used to separate the pixels of an input image into several classes by
+  maximizing the variances between classes --
+  `skimage.filters.threshold_multiotsu` (#3872, #4174)
+- Added new example data -- `skimage.data.shepp_logan_phantom`, `skimage.data.colorwheel`,
+  `skimage.data.brick`, `skimage.data.grass`, `skimage.data.roughwall`, `skimage.data.cell`
```

Comment:
I don't think these datasets are new; we just replaced them with new versions that didn't have dubious licensing.

Reply:
@stefanv no, they are exposed in `data` for the first time. Before, we had them but they were just used in testing.